layouts/index.html

```diff
@@ -1,12 +1,11 @@
 {{ define "main" }}
   {{- partial "intro.html" . -}}
-  <div class="container p-6 mx-auto grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4 lg:gap-8">
```

Comment:
Looks like this line with a `div` was removed without any reason. Can you revert this?

Reply:
Yes, that went wrong somehow.